### PR TITLE
Pre-Develop Changes

### PR DESCRIPTION
**Pull request by amosh101**

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/ResumoMensalCohortQueries.java [hunk 1]
```diff
@@ -76,7 +76,7 @@ public CohortDefinition getNumberOfPatientsWhoInitiatedPreTarvByEndOfPreviousMon
         "A1I",
         map(
             getNumberOfPatientsInMasterCardWithArtLessThanStartDateA1(),
-            "startDate=${startDate},location=${location}"));
+            "startDate=${startDate-1},location=${location}"));
```
Comment (ningosi) [inline api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/ResumoMensalCohortQueries.java]: Parameter manipulations are NOT done this way, you have to specify what arithmetic will need to work with, please include m for months, y for year, d for days etc.

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/ResumoMensalCohortQueries.java [hunk 2]
```diff
@@ -86,12 +86,12 @@ public CohortDefinition getNumberOfPatientsWhoInitiatedPreTarvByEndOfPreviousMon
         "A1III",
         map(
             getAllPatientsEnrolledInPreArtProgramWithDateEnrolledLessThanStartDateA1(),
-            "startDate=${startDate},location=${location}"));
+            "startDate=${startDate-1},location=${location}"));
```
Comment (ningosi) [inline api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/ResumoMensalCohortQueries.java]: Same here; date parameters need to well done

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/ResumoMensalCohortQueries.java (hunk 1 above)
Comment (amosh101) [reply api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/ResumoMensalCohortQueries.java]: Updated

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/ResumoMensalCohortQueries.java (hunk 2 above)
Comment (amosh101) [reply api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/ResumoMensalCohortQueries.java]: Updated as well

**Review by ningosi — COMMENTED** [review]
Looks fine in terms of code structure. I need to build and make sure it runs fine before merging it

**Pull request merged**